index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JoseMontoya.com</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header>
+        <div class="status_bar">
+            <p>9:41</p>
+            <img src="./assets/status_bar_icons.svg" alt="phone icons">
+        </div>
+        <div class="tool_bar">
+            <h1>Jose Montoya</h1>
+            <img src="./assets/menu_icon.svg" alt="menu">
+        </div>
+    </header>
+    <section class="headline">
+        <h2>Hey There. <br/> I'm Jose</h2>
+
+        <h3>I am a Software Developer</h3>
+
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.
+        </p>
+
+        <img class="social_media" src="./assets/social_media_icons.svg" alt="social media icons">
+        
+        <img class="background_image" src="./assets/background_image.svg" alt="">
```

Comment:
- I can suggest you use the background image in the CSS so that it will not affect  the still of your app by adding some extra space and removing the scrolling button from left to right and top to bottom as shown in the picture below make sure the body of your app takes the max screen without scrolling button  :+1: 

![image](https://user-images.githubusercontent.com/95297251/204928014-0aab2cd2-38cb-4897-ac8d-daa24b3f2e64.png)